app.js

```diff
@@ -22,40 +26,34 @@ app.get('/__gtg', (req, res) => {
 // app.get('/__health', healthChecks);
 
 app.get('/', (req, res) => {
-	const reqData = {
-		url: req.url,
-		method: req.method,
-		headers: req.headers
-	}
-	logger.info({ event: 'ENDPOINT_VISIT', data: reqData})
 	res.json('ip events service v2: 2 events 2 furious');
 });
 
-if (env !== 'test') {
-	app.use(require('./src/middleware/api-key'));
+if (env === 'prod') {
```

Comment:
Small thing but I think NODE_ENV is usually seen as `production` rather than `prod`

Reply:
Will update! 👍